spec/controllers/wishlists_controller_spec.rb

```diff
@@ -4,7 +4,7 @@
   describe "POST #create" do
     context "successfully creates new wishlist" do
```

Comment:
It probably doesn't make sense to make all these changes as part of this PR, but IMO the context should specify the conditions ("given xyz" or "when abc") so we'd have something like "given valid parameters" it "creates a new wishlist and redirects to dashboard" or something.